src/compressed_tensors/compressors/quantized_compressors/base.py

```diff
@@ -145,6 +147,17 @@ def compress(
 
         return compressed_dict
 
+    def _check_if_zp_pack_quantized(self, quant_args):
+        from compressed_tensors.compressors import PackedQuantizationCompressor
+
+        if isinstance(self, PackedQuantizationCompressor):
+            if not quant_args.symmetric and quant_args.strategy in [
+                QuantizationStrategy.GROUP.value,
+                QuantizationStrategy.CHANNEL.value,
+            ]:
+                return True
+        return False
+
```

Comment:
not sure if this looks any better 😅 
```suggestion
        return (
            isinstance(self, PackedQuantizationCompressor)
            and (not quant_args.symmetric)
            and (
                quant_args.strategy
                in [
                    QuantizationStrategy.GROUP.value,
                    QuantizationStrategy.CHANNEL.value,
                ]
            )
        )

```

Reply:
I kinda like mine better lmao